README.md

```diff
@@ -37,4 +37,5 @@ Feel free to check the issues page.
 
 etc
 📝 License
+
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you update your **README** file syntax to make it looks smart and cleaner e.g.: 
  - the **Getting Started** is supposed to be a title and not a **technology** used to build the app.
  - Add command on how to clone the repo, access the cloned directory, run the app locally after it is cloned, install dependencies(if available),...
  - Add to make your deployed link looks smart you can copy this text below as it is(this syntax will show text inside the square brackets as a link):
```
e.g.:
[Here is the Live Demo of My Porfolio](https://mishdm.github.io/portfolio-page/)
```
 - you can use this[ template to help you update your README](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md) (you can copy all the content by clicking the **[Copy raw contents](https://github.blog/changelog/2021-09-20-quickly-copy-the-contents-of-a-file-to-the-clipboard/)** button or click the `Raw` button to see all the semantics inside the file) then edit your README file according to your information.